### PR TITLE
fix pep8 W503

### PR DESCRIPTION
**Pull request by gurneyalex**

### CODE REVIEW

**Comment by lepistone** [comment]
:+1:

**Comment by lepistone** [comment]
why "in progress" @nhomar ?

File: database_cleanup/model/purge_tables.py [hunk 1]
```diff
@@ -110,9 +110,9 @@ def find(self, cr, uid, context=None):
             known_tables += [
                 column._sql_names(model_pool)[0]
                 for column in model_pool._columns.values()
-                if column._type == 'many2many'
+                if column._type == 'many2many' and
                 # unstored function fields of type m2m don't have _rel
-                and hasattr(column, '_rel')
+                hasattr(column, '_rel')
```
Comment (yvaucher) [inline database_cleanup/model/purge_tables.py]: I think this is less readable

Comment (yvaucher) [inline database_cleanup/model/purge_tables.py]: What about:

```
known_tables += [
    column._sql_names(model_pool)[0]
    for column in model_pool._columns.values()
    if (column._type == 'many2many' and
        # unstored function fields of type m2m don't have _rel
        hasattr(column, '_rel'))
]
```

**Comment by lepistone** [comment]
@yvaucher for me it's :+1: both ways.

**Comment by gurneyalex** [comment]
@yvaucher do you like it better this way?

**Comment by yvaucher** [comment]
@gurneyalex Thanks for the change :+1:

**Comment by yvaucher** [comment]
I fast track it

**Pull request merged**